packages/forEach/src/forEachModule.ts

```diff
@@ -0,0 +1,23 @@
+import {IConstraintModule, ConstraintValue, ConstraintExecuter, ConstraintConfiguration, IRuntimeContext, IValidator} from '@configurable-validator/core'
+
+
+export class ForEachModule implements IConstraintModule {
+
+    private _validator: IValidator;
+
+    initialize(validator: IValidator) {
+        this._validator = validator;
+    }
+
+    buildConstraintExecuter(value: ConstraintValue, externalData?: any): ConstraintExecuter {
+
+        const config = value as ConstraintConfiguration;
```

Comment:
add validation guard for this

Reply:
actually this is a weird case. You can't really type guard interfaces in typescript (i.e. you can't use `instanceof`).